app/models/developer_external_profiles/linkedin_profile_fetcher.rb

```diff
@@ -0,0 +1,31 @@
+module DeveloperExternalProfiles
+  class LinkedinProfileFetcher
+    def conversation_developer_profiles
+      developer_external_profiles_list = []
+
+      Conversation.includes(:developer).where.not(developers: {linkedin: nil}).map(&:developer).uniq.each do |developer|
```

Comment:
We should be able to query for developers directly instead of mapping and uniquing.

`Developer.joins(:conversation).where.not(linked_in: [nil, ""]).distinct`